service/src/main/resources/config/aou/test/SC2023Q3R1/entitygroup/t_zipcode_socioeconomic_person.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "t_zipcode_socioeconomic_person",
+  "type": "GROUP_ITEMS",
+  "groupEntity": "t_person",
+  "itemsEntity": "t_zipcode_socioeconomic",
+  "sourceDataMapping": {
+    "dataPointer": "omop_dataset",
+    "relationshipMappings": {
+      "groupToItems": {
+        "idPairsTable": { "rawSqlFile": "t_zipcode_socioeconomic.sql" },
+        "idPairsIdA": { "column": "person_id" },
+        "idPairsIdB":  { "column": "person_id" }
```

Comment:
This doesn't look right. The entity group definitions are basically defining JOINs between tables, so it doesn't make sense to join the same two fields on each other.

Reply:
Not sure what is correct. I just need the [zipcode sql ](https://github.com/DataBiosphere/tanagra/blob/19af2091a9f745db3685d10c12356407b94d6a81/service/src/main/resources/config/aou/test/SC2023Q3R1/sql/t_zipcode_socioeconomic.sql#L1-L1) table to be a prepackaged concept setup.

Reply:
Updated t_zipcode_socionomic tables and SQL